src/main/webapp/WEB-INF/views/UserPage.html

```diff
@@ -0,0 +1,384 @@
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <a href="#favourites" class="btn btn-info btn-block" data-toggle="collapse">Favourites</a>
+            <div class="collapse" id="favourites">
+                <div class="well">
+                    <div class="panel-info">
+                        <div class="panel-body">
+                            <div class="row masonry" data-columns>
+                                <div class="card">
+                                    <a href="#"><img class="card-img-top img-responsive"
+                                                     src="https://goo.gl/bLja0p"></a>
+                                    <div class="card-block">
+                                        <a href="#"><h4 class="card-title">The NOMATIC Backpack and Travel Pack</h4></a>
+                                        <div class="card-text">
+                                            <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                                                tempor
+                                                incididunt ut labore et
+                                                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                                                ullamco
+                                                laboris nisi ut aliquip
+                                                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                                                voluptate
+                                                velit
+                                                esse cillum dolore eu
+                                                fugiat nulla pariatur.
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="card-footer">
+                                        <div class="row">
+                                            <div class="col-xs-6">
+                                                <div>min</div>
+                                                <div><i class="fa fa-dollar"></i>100000</div>
+                                            </div>
+                                            <div class="col-xs-6">
+                                                <div class="right-box">Still needed</div>
+                                                <div class="right-box"><i class="fa fa-dollar"></i>9999</div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <a href="#"><img class="card-img-top img-responsive"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)